src/curves.rs

```diff
@@ -594,6 +598,27 @@ macro_rules! new_curve_impl {
             }
         }
 
+        impl group::cofactor::CofactorCurveAffine for $name_affine {
```

Comment:
At some point we can probably just add a blanket derive for this on `T: PrimeCurveAffine`, since the impl is trivial.